openpype/hosts/resolve/hooks/pre_resolve_launch_last_workfile.py

```diff
@@ -35,6 +37,7 @@ def execute(self):
         key = "OPENPYPE_RESOLVE_OPEN_ON_LAUNCH"
         self.launch_context.env[key] = last_workfile
 
+    def set_startup_script(self):
```

Comment:
Seeing this being done I actually have the feeling that this logic should not at all be part of e launch hook named "ResolveLaunchLastWorkfile"? It's related to that, somewhat, but also does more than just that? Right?

Reply:
Good point. I will separate it from it.